### PR TITLE
Saving positions for epubs, pdfs, and video/audio

### PR DESCRIPTION
**Pull request by christianmemije**
### Summary

Meant to address  #4268 and #4243
…

### Reviewer guidance

Test that video player settings are saved for: 
* Epubs
* PDFs
* Videos

**Should I be force saving?**

…

### References

----

### Contributor Checklist

- [ ] Contributor has fully tested the PR manually
- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

File: kolibri/plugins/document_epub_render/assets/src/views/EpubRendererIndex.vue [hunk 1]
```diff
@@ -457,6 +447,32 @@
...
+
+        // this is not working, hence the delay via the debounce
+        // this.resizeRendition(width, height);
+        this.debounceResizeRendition(width, height);
+
+        this.rendition.on(EVENTS.RENDITION.RELOCATED, location => this.relocatedHandler(location));
+        this.rendition.on('keyup', this.handleKeyUps);
+        this.rendition.on('click', () => this.closeSideBar());
+
+        window.addEventListener('mousedown', this.handleMouseDown, true);
```
Comment (rtibbles) [inline kolibri/plugins/document_epub_render/assets/src/views/EpubRendererIndex.vue]: Is this meant to be a passive listener? If so, should be `{ passive: true }` as the final arg.

**Comment by indirectlylit** [comment]
> Should I be force saving?

sorry, what do you mean by this precisely?

**Comment by rtibbles** [comment]
Shouldn't need to be passing a force param to the save, no.

@indirectlylit the log saving actions have a force param to override the time/progress based buffering that only lets logs get saved when a certain interval has been passed.

**Comment by indirectlylit** [comment]
@christianmemije would you please add some basic Gerkhin stories that reflect this new behavior so we ensure that it's covered during integration testing?

**Comment by indirectlylit** [comment]
I tested with videos and PDFs. PDF works very well, but videos do not seem to save state.

It's also not clear to me what state is expected to be saved: e.g. speed of video playback? PDF zoom level? Epub text size? etc. Let's document these in the Gherkins.

**Review by indirectlylit — APPROVED** [review]
this is working well enough to merge in. Filing follow-up issue here: #4367

**Pull request merged**